extended.js

```diff
@@ -0,0 +1,47 @@
+// Fizz Buzz++
```

Comment:
We should not commit compiled files (.js in this case) as this is redundant and may be different from machine to machine (i.e. on UNIX will have different line endings)